### ISSUE TITLE
Wrong mechanics results.

### ISSUE THREAD

**Issue by Paulie14** [flow123d/flow123d#1345]
I tested a very simple loading problem in mechanical model in master which displays very weird behaviour.
I created a branch `PE_bc_mechanics` and pushed there the test 50/30.
It is a box with fixed bottom, it is loaded on top (z axis), and normal displacement is restricted on the sides.

I also computed the problem in branch `JS_bc_mechanics` which gives qualitatively satisfying results and used it as reference data.

I dug into the assembly in elasticity and found the following:
Although the `traction` boundary condition is prescribed on `.top_z` boundary, in the assembly the `bc_traction` field is non-zero only on element 588. I have no idea why...

Add following prints at `assembly_elasticity.hh:430` to see it:
```
DebugOut() << "elm_idx: " << cell_side.cell().elm().index();
            for (auto p : this->boundary_points(cell_side) )
            {
                for (unsigned int i=0; i<n_dofs_; i++){
                    double val = eq_fields_->cross_section(p) *
                            arma::dot(vec_view_bdr_->value(i,k), eq_fields_->bc_traction(p) + eq_fields_->potential_load(p) * fe_values_bdr_side_.normal_vector(k)) *
                            fe_values_bdr_side_.JxW(k);
                    local_rhs_[i] += val;
                    if(std::abs(val) > 0)
                        DebugOut() << "elm_idx: " << cell_side.cell().elm().index() << "   local_rhs_[" << i << "] +=" << val << "\n";
                    if(std::abs(eq_fields_->bc_traction(p)[2]) > 0)
                        DebugOut() << "cs: " << eq_fields_->cross_section(p) << "  traction: " << eq_fields_->bc_traction(p)[2] << " JxW: " << fe_values_bdr_side_.JxW(k) << "\n";
                }
                ++k;
            }
```

**Comment by dflanderka** [comment]
Correct loop at `assembly_elasticity.hh:430`:
```
            for (auto p : this->boundary_points(cell_side) )
            {
            	auto p_bdr = p.point_bdr( side.cond().element_accessor() );
                for (unsigned int i=0; i<n_dofs_; i++)
                    local_rhs_[i] += eq_fields_->cross_section(p) *
                            arma::dot(vec_view_bdr_->value(i,k), eq_fields_->bc_traction(p_bdr) + eq_fields_->potential_load(p) * fe_values_bdr_side_.normal_vector(k)) *
                            fe_values_bdr_side_.JxW(k);
                ++k;
            }
```

**Comment by Paulie14** [comment]
Fixed and tested by 298a1b496ed8949b998fc136dc1a67387b541ee7.
